acme/setup.py

```diff
@@ -53,7 +53,7 @@
     version=version,
     description='ACME protocol implementation in Python',
     url='https://github.com/letsencrypt/letsencrypt',
-    author="Let's Encrypt Project",
+    author="Electronic Frontier Foundation",
```

Comment:
Perhaps. The client was written almost exclusively by you and EFF employees and with the client becoming an EFF project, this seemed fitting to me.

With that said, I don't have too strong of an opinion on the issue. What do you think @pde?